### PR TITLE
Adding new Purple haze themes with rounded layout

### PR DESCRIPTION
**Pull request by grafixeyehero**
Screenshot_2019-04-18 Jellyfin(3)

### CODE REVIEW

File: src/components/themes/purple-haze/theme.css [hunk 1]
```diff
@@ -0,0 +1,476 @@
...
+    color: inherit
+}
+
+.checkboxListLabel,
+.inputLabel,
+.inputLabelUnfocused,
+.paperListLabel,
+.textareaLabelUnfocused {
+    color: #bbb;
+    color: rgba(255, 255, 255, .7)
```
Comment (DrPandemic) [inline src/components/themes/purple-haze/theme.css]: I guess having 2 `color` is a typo?

Comment (grafixeyehero) [reply src/components/themes/purple-haze/theme.css]: Not all browsers support RGBa, so if the design permits, you should declare a "fallback" color. This color will be most likely be solid (fully opaque). Not declaring a fallback means no color will be applied in browsers that don't support it. This fallback does fail in some really old browsers.

**Comment by anthonylavado** [comment]
I still really like this, so unless there's objection from @jellyfin/web...

**Comment by grafixeyehero** [comment]
@thornbill 
Screenshot_2019-04-25 Jellyfin

**Pull request merged**